modules/ecs/variables.tf

```diff
@@ -0,0 +1,499 @@
+variable "private_subnets" {
+  description = "(Optional) List of private subnet IDs, if not specified then the subnets listed in the private_subnets_to_create variable will be created and used"
+  type        = list(string)
+  default     = []
+}
+
+variable "private_subnets_to_create" {
+  description = "(Optional) A map of subnet objects to create"
+  type = list(object({
+    cidr_block = string
+    az         = string
+    public     = bool
+  }))
+  default = [
+    {
+      cidr_block = "10.0.1.0/24"
+      az         = "default"
+      public     = false
+    },
+    {
+      cidr_block = "10.0.2.0/24"
+      az         = "default"
+      public     = false
+    },
+    {
+      cidr_block = "10.0.3.0/24"
+      az         = "default"
+      public     = false
+    }
+  ]
+}
```

Comment:
these defaults are already defined in the root variables and passed in, so can be removed from this file

Reply:
Removed